joblib/test/test_func_inspect.py

```diff
@@ -72,6 +42,21 @@ def k(*args, **kwargs):
     pass
 
 
+@fixture(scope='module')
+def f2(tmpdir_factory):
+    # Create a Memory object to test decorated functions.
+    # We should be careful not to call the decorated functions, so that
+    # cache directories are not created in the temp dir.
```

Comment:
I have to say I don't really understand this comment but it was there before ...